back-end/helpers/logger.js

```diff
@@ -0,0 +1,49 @@
+const { createLogger, format, transports } = require('winston');
```

Comment:
This creates a singleton logger instance that can be `required` throughout our project and used to add logs.